docs/platforms/php/guides/symfony/index.mdx

```diff
@@ -92,3 +92,13 @@ class SentryTestController extends AbstractController
 ```
 
 After you visit the `/_sentry-test` page, you can view and resolve the recorded error by logging into [sentry.io](https://sentry.io) and opening your project. Clicking on the error's title will open a page where you can see detailed information and mark it as resolved.
+
+## Local Development and Testing
+
+When Sentry is installed in your application, it will also be active when you are developing or running tests. Because PHP is single-threaded, Sentry will send events synchronously, which can slow down your application or test suite.
+
+You most likely don't want errors to be sent to Sentry when you are developing or running tests so they don't use up [quota](/pricing/quotas/). To avoid this, set the DSN value to `null` to disable sending errors to Sentry.
```

Comment:
fyi, I've seen this trip up AI when answering "How to disable errors but keep performance?" because it sounds like setting DSN to `null` disables errors specifically.

Reply:
Do we really need to optimize our docs for an AI system that seems to not work properly?